pkg/util/log/clog.go

```diff
@@ -351,7 +359,7 @@ func (d *EntryDecoder) Decode(entry *Entry) error {
 			return io.EOF
 		}
 		b := d.scanner.Bytes()
-		m := entryRE.FindSubmatch(b)
+		m := entryCaptureRE.FindSubmatch(b)
```

Comment:
We could strip the prefix, yeah, but then as far as I can tell we'd have to run the regex twice here (or the regex once and some other logic once) -- once to to capture the groups and once to figure out where the message starts since otherwise the length of the matched portion isn't known.

I've gone with your second suggestion and removed the message part from `entryMatchRE`.